game/game.go

```diff
@@ -708,3 +709,18 @@ func (g *Game) playActivatedAbility() {
 	g.Print()
 	panic("playActivatedAbility failed")
 }
+
+// Returns a binary representation of the game, json for now
```

Comment:
It's probably worth trying other methods whenever speed becomes bothersome. This https://github.com/alecthomas/go_serialization_benchmarks seems to recommend gogoprotobuf